js/client-teambuilder.js

```diff
@@ -1245,6 +1254,39 @@
 			this.save();
 		},
 
+		setNotesBlur: function (e) {
+			var textarea = $(e.currentTarget);
+			if (!$.trim(textarea.val())) textarea.parent().hide();
+			if (textarea.val().indexOf('|') >= 0 || textarea.val().indexOf(']') >= 0 || textarea.val().indexOf('\t') >= 0) {
+				app.addPopupMessage("Unfortunately, set notes can't contain tabs or the characters | and ], since they're used for storing teams.");
```

Comment:
I have to admit, this is the first I hear of it expiring on its own. The specification says 
>User agents should expire data from the local storage areas only for security reasons or when requested to do so by the user.

Is there some security reason behind this?
Is there some other way to store these notes? I assume not, but I don't want this (very useful) feature to get shelved over this :(